SyntheticsRunTestsTask/task.json

```diff
@@ -116,6 +116,27 @@
       "label": "Polling Timeout",
       "defaultValue": "",
       "helpMarkDown": "The duration (in milliseconds) after which the task stops polling for test results."
+    },
+    {
+      "name": "failOnCriticalErrors",
+      "type": "boolean",
+      "label": "Fail on critical errors",
+      "defaultValue": "",
+      "helpMarkDown": "Fail the CI job if no tests were triggered, or results could not be fetched from Datadog."
+    },
+    {
+      "name": "failOnMissingTests",
+      "type": "boolean",
+      "label": "Fail on missing tests",
+      "defaultValue": "",
+      "helpMarkDown": "Fail the CI job if at least one specified test with a public ID (a `--public-id` CLI argument or listed in a [test file](https://docs.datadoghq.com/continuous_testing/cicd_integrations/configuration/?tab=npm#test-files)) is missing in a run (for example, if it has been deleted programmatically or on the Datadog site)."
```

Comment:
@alai97 for azure devops, you can see that markdown is supported, and the help is shown like this:

![image](https://github.com/DataDog/datadog-ci-azure-devops/assets/9317502/c3873f2f-81e5-4868-9f37-92d2cb987888)